CWE/history/countries/NNG - Northern Nigeria.txt

```diff
@@ -0,0 +1,96 @@
+capital = 1948
+primary_culture = hausa
+religion = secularism
+government = hms_government
+plurality = 5.0
+nationalvalue = nv_equality
+literacy = 0.01
+non_state_culture_literacy = 0.0
+civilized = yes
+prestige = 0
+last_election = 1944.2.1
+ruling_party = "Jamiyaar Mutanem Arewa (R)"
+upper_house = {
+	populist = 0
+	liberal = 5
+	conservative = 45
+	traditionalist = 40
+	progressive = 10
+	socialist = 0
+	communist = 0
+	}
+# Political Reforms
+political_opinions = regulated_opinion
+minorities_reform = minorities_oppression
+immigration_policy = quota_immigration
+civil_service = politicised_service
+political_parties = parties_only_nominated
+lgbt_rights = lgbt_outlawed
+publishing_rights = censored_publishing
+union_rights = no_trade_unions
+voting_system = first_past_the_post
+vote_franschise = registered_voting
+public_meetings = limited_meetings penal_system = capital_punishment
+# Social Reforms
+safety_regulations = some_regulations
+pensions = no_pensions
+education = self_financed_education
+health_care = no_health_care
+security_reform = police
+child_care = no_child_care
+unemployment_subsidies = no_subsidies
+pollution = no_controls
+wage_and_work_hours = unregulated_hours
+social_service_reform = no_social_service
+# Technologies
+# Starting Consciousness
+consciousness = 0
+nonstate_consciousness = 0
+schools = army_tech_school
+
+1992.1.1 = {
+	capital = 1948
+	religion = secularism
+	government = theocracy
+	plurality = 19.0
+	nationalvalue = nv_equality
+	literacy = 0.09
+	non_state_culture_literacy = 0.045
+	civilized = yes
+	prestige = 9
+	ruling_party = "Yan Tatsine (Islamist)"
+	upper_house = {
+		populist = 4
+		liberal = 21
+		conservative = 0
+		traditionalist = 45
+		progressive = 0
+		socialist = 25
+		communist = 5
+		}
+	# Political Reforms
+	political_opinions = regulated_opinion
+	minorities_reform = minorities_equal_rights
+	immigration_policy = quota_immigration
+	civil_service = politicised_service
+	political_parties = parties_only_nominated
+	lgbt_rights = lgbt_outlawed
+	publishing_rights = censored_publishing
+	union_rights = no_trade_unions
+	voting_system = first_past_the_post
+	vote_franschise = registered_voting
+	public_meetings = limited_meetings penal_system = capital_punishment
+	# Social Reforms
+	safety_regulations = some_regulations
+	pensions = no_pensions
+	education = self_financed_education
+	health_care = no_health_care
+	security_reform = police
+	child_care = no_child_care
+	unemployment_subsidies = no_subsidies
+	pollution = no_controls
+	wage_and_work_hours = unregulated_hours
+	social_service_reform = no_social_service
+	# Technologies
+oob = "1992/NGE_oob.txt"

```

Comment:
Cannot have OOB entry in countries not existing at the start as that can cause a crash when the player clicks on the release nation tab if this country is a possible vassal of THIS.

Reply:
I swear github just does whatever it wants sometimes. Its like HAL from 2001.